lib/permutations.rb

```diff
@@ -1,4 +1,24 @@
+# Write a method which will take two strings as arguments and returns true if one string is a permutation of the other.
+#permutations?("hello", "ehllo") => true
+#permutations?("Pizza", "Pasta") => false
+#permutations?("", "") => true
 
 def permutations?(string1, string2)
+  hash_check = {}
+  string1.chars.each do |letter|
+    hash_check[letter] = true
+  end 
```

Comment:
You're not counting the number of each letter, so "too", and "tot" would be considered permutations.